devtools/server/actors/replay/connection.js

```diff
@@ -759,9 +761,8 @@ function handleRecordingStarted(pmm) {
     // Log the reason so we can see in our CI logs when something went wrong.
     console.error("Unstable recording: " + data.why);
     const browser = getBrowser();
-    const key = getRecordingKey(browser);
 
-    setRecordingFinished(browser, `https://replay.io/browser/error?message=${data.why}`);
+    const key = setRecordingFinished(browser, `https://replay.io/browser/error?message=${data.why}`);
```

Comment:
Returning the key from `setRecordingFinished` seems out of place to me given its current name. Would moving the `getRecordingKey()` after this line produce the same result?

Reply:
Good call hahah.